CONTRIBUTING.md

```diff
@@ -49,3 +49,15 @@ Now several cases are possible:
    Choose the start branch (the one you want to send your changes to) and the end branch (the one you want to send your changes to).
    Put an understandable name, click on `Assign Yourself` on the right,
    At the bottom of the description, note the id of your github issue like this: `Related issues: #1` or `Close #1`.
+
+## Create new Release
+
+To publish a new release you need to the repository, you need to be sure to be on the latest version. so you can use this commad : 
+
+```sh
+git checkout master && git pull && npm version minor
```

Comment:
I suggest to wrap these commands into a npm script:
```jsonc
// package.json
{
    "scripts": {
        "release": "git checkout && npm version"
    }
}
```

To release a new version just simply run:
```sh
# npm run release -- %version%
> npm run release -- minor
```
Double dash is used to separate arguments. Everything is followed by `--` will be applied to the actual command in the script (`npm` in this case, as it's the last one).

Ref: https://docs.npmjs.com/cli/run-script

Reply:
This addition looks good for me. 
Thanks.